base/some.jl

```diff
@@ -145,3 +145,11 @@ macro something(args...)
     something = GlobalRef(Base, :something)
     return :($something($expr))
 end
+
+==(a::Some, b::Some) = a.value == b.value
+==(::Some{Missing}, ::Some{T}) where T = T === Missing
+==(::Some{T}, ::Some{Missing}) where T = T === Missing
```

Comment:
```suggestion
==(::Some{Missing}, ::Some) = false
==(::Some, ::Some{Missing}) = false
```
Unless I'm missing something, these should both be `false` regardless of what `T` is, since if `T === Missing` then neither of these methods would even be called, as `==(::Some{Missing}, ::Some{Missing})` applies and is more specific.

Reply:
Yeah, just a leftover from the first commit without the method handling the ambiguity.